### PR TITLE
CRS-158: TypingIndicator refactor

### PR DESCRIPTION
**Pull request by jaapbakker88**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: src/components/TypingIndicator/TypingIndicator.js [hunk 1]
```diff
@@ -48,9 +49,10 @@ const TypingIndicator = (props) => {
 
 TypingIndicator.propTypes = {
   /** @see See [chat context](https://getstream.github.io/stream-chat-react/#chatcontext) doc */
+  // @ts-ignore
   client: PropTypes.object,
```
Comment (vishalnarkhede) [inline src/components/TypingIndicator/TypingIndicator.js:53]: We want to move this to useContext? Or will it be in different PR?

Comment (jaapbakker88) [reply src/components/TypingIndicator/TypingIndicator.js:53]: yeah that probably made sense, hadn't figured that out yet when I created this pr

Comment (jaapbakker88) [reply src/components/TypingIndicator/TypingIndicator.js:53]: this component is not actually using the context

Comment (jaapbakker88) [reply src/components/TypingIndicator/TypingIndicator.js:53]: just get's passed as props
